spec/controllers/articles_controller_spec.rb

```diff
@@ -1,47 +1,75 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
-    describe 'show the list to articles saved' do
+  describe 'GET #index' do
+    it 'renders the index template' do
+      get :index
+      expect(response).to render_template(:index)
     end
   end
 
-  describe '#show' do
-    it 'show the articles' do
+  describe 'GET #show' do
+    let(:article) { FactoryGirl.create(:article) }
+
+    it 'assigns the article with a given id' do
+      get :show, id: article.id
+      expect(assigns(:article)).to eq(article)
+    end
+
+    it 'renders the article view' do
+      get :show, id: article.id
+      expect(response).to render_template(:show)
+    end
+  end
+
+  describe 'GET #new'do
+    let(:article) { FactoryGirl.build(:article) }
+
+    it 'assigns an article' do
+      get :new
+      expect(assigns(:article)).to be_a_new(Article)
+    end
+  end
+
+  describe 'POST #create' do
+    let(:article) { FactoryGirl.create(:article)}
+
+    it 'renders the new artice' do
+      post :create, title: article.title
+                  , text: article.text
+      expect(render(:article)).to be_a_new(Article)
+    end
+    it 'render the article view' do
+      get :new, id: article.id
+      expect(response).to render_template(:new)
     end
   end
 
-  describe '#new'do
-    it 'create new post' do
+  describe 'PUT #update' do
+    let(:article), { FactoryGirl.build(:article) }
+
+    it 'update the article with a given id' do
+      put :update, id: article.id
+      expect(response).to eq(article)
+    end
+    it 'finds the requested article' do
+      put :update, title: article.title
```

Comment:
Lines length are good enough when they have 80 columns of length - `clean code` -, so here is unnecessary break this line. But it is a personal opinion. I feel comfortable with less than 100-120 columns length.
when you break a line, the `,` should be in the previous line:

```
put :update, title: article.title,
             text: article.text
```